persistence.js

```diff
@@ -491,7 +513,7 @@ RedisPersistence.prototype.streamWill = function (brokers) {
 }
 
 RedisPersistence.prototype.getClientList = function (topic) {
-  var clientIds = this._matcher.match(topic).map(toClientIds)
+  var clientIds = this._matcher.match(topic).filter(isTopic).map(toClientIds)
```

Comment:
If a client is also subscribed to a wildcard which matches the topic, won't two matches be returned?